Don't merge - componentName (#86)
decentraland/documentation: content/creator/sdk7/architecture/custom-components.md

```

## About the componentName

> Each component must have a unique componentName, that differentiates it internally. You won't need to use this internal identifier anywhere else in your code. A good practice is to use the same name you assign to the component, but starting with a lower case letter, but all that really matters is that this identifier is unique within the project.

Reviewer: ```suggestion
Each component must have a unique component name or identifier, that differentiates it internally. You won't need to use this internal identifier anywhere else in your code. A good practice is to use the same name you assign to the component, but starting with a lower case letter, but all that really matters is that this identifier is unique within the project.

<!-- UNCOMMENT WHEN EDITOR IS READY TO BE RELEASED
It is important to notice that this component name should never change, so pick it carefully. It is used to store the composites (a.k.a. prefabs) in a human readable format.
-->
```

Every field in the schema must include a type declaration. You can only use the special schema types provided by the SDK. For example, use the type `Schemas.Boolean` instead of type `boolean`. Write `Schemas.` and your IDE will display all the available options.

```ts

> export const WheelSpinComponent = engine.defineComponent("wheelSpinComponent", {

Reviewer: ```suggestion
export const WheelSpinComponent = engine.defineComponent("WheelSpinComponent", {
```

```ts
// Option 1: Inline definition

> export const WheelSpinComponent = engine.defineComponent("wheelSpinComponent", {

Reviewer: ```suggestion
export const WheelSpinComponent = engine.defineComponent("WheelSpinComponent", {
```

}

//// component

> export const WheelSpinComponent = engine.defineComponent("wheelSpinComponent", mySchema)

Reviewer: ```suggestion
export const WheelSpinComponent = engine.defineComponent("WheelSpinComponent", mySchema)
```

  speed: Schemas.Float,
}

> export const LerpTransformComponent = engine.defineComponent("lerpTransformComponent", MoveTransportData)

Reviewer: ```suggestion
export const LerpTransformComponent = engine.defineComponent("LerpTransformComponent", MoveTransportData)
```

```ts
// define component

> export const WheelSpinComponent = engine.defineComponent("wheelSpinComponent", {

Reviewer: ```suggestion
export const WheelSpinComponent = engine.defineComponent("WheelSpinComponent", {
```
